src/Microsoft.VisualStudio.Editors/PropPages/ApplicationPropPage.vb

```diff
@@ -611,6 +611,10 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
                         If Trim(Win32ResourceFile.Text).Length = 0 Then
                             message = My.Resources.Microsoft_VisualStudio_Editors_Designer.PropPage_NeedResFile
                             Return ValidationResult.Warning
+                        ElseIf Not Win32ResourceFile.Text.IndexOfAny(Path.GetInvalidPathChars()) = -1 Then
+                            Dim FirstInvalidCharacter As String = Path.GetInvalidPathChars().First
+                            message = My.Resources.Microsoft_VisualStudio_Editors_Designer.PropPage_InvalidCharactersInFilePath & vbCrLf & "Please remove the following character: " & FirstInvalidCharacter
```

Comment:
I don't think this does what you think it does. This will always show a double quote to the user, as that is what is first in `Path.GetInvalidChars()`. What you probably want to do is get the value of `Win32ResourceFile.Text.IndexOfAny(Path.GetInvalidPathChars())` before the `If` block, and then if that value is not -1, add it to the message via `Win32ResourceFile.Text.Substring(theCharacter, 1)`.

Reply:
Fixed this, thanks!